src/storage/s3v4OS.py

```diff
@@ -401,7 +401,7 @@ def create_upload_form(self, sourcepath, containername, prefix, ttl, max_file_si
             "headers": {}
         }
 
-        command = f"curl -i -X {httpVerb} {endpoint_url}/{containername}"
+        command = f"curl -s -i -X {httpVerb} {endpoint_url}/{containername}"
```

Comment:
Would it not be better to use both "--silent --show-error" options, so that it does not write status update to stderr but it does show an error in case of a problem?

Reply:
I've tested that and it doesn't seem to show any output in stderr if the command was executed correctly. I will push changes in this branch (dev).